oshmem/runtime/runtime.h

```diff
@@ -237,6 +237,20 @@ OSHMEM_DECLSPEC int oshmem_shmem_register_params(void);
 
 #endif  /* OSHMEM_PARAM_CHECK */
 
+#define RUNTIME_SHMEM_NON_IMPLEMENTED_API_ABORT()                    \
+    do {                                                             \
+        SHMEM_API_ERROR("Called non-implemented API: %s", __func__); \
+        oshmem_shmem_abort(OSHMEM_ERR_NOT_IMPLEMENTED);              \
+    } while (0)
+
+#define RUNTIME_SHMEM_NON_IMPLEMENTED_API_ABORT_RET_SIZE_T() \
```

Comment:
If the `return` is just for compilation (no meaning for the code), maybe have a single macro for both types?